### PR TITLE
Apache Deltaspike Data Module With JAXRS Wildfly Fraction

### PR DESCRIPTION
**Pull request by rafaelszp**
As Bob McWhirter requested I created a simple example using deltaspike data with wildfly swarm using jpa, jaxrs and datasource fractions.

### CODE REVIEW

File: jaxrs/jaxrs-deltaspike-data/pom.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        </plugins>
+    </build>
+
+    <dependencies>
+        <!-- Java EE 7 dependency -->
+        <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-api</artifactId>
+            <version>7.0</version>
+            <scope>provided</scope>
```
Comment (bobmcwhirter) [inline jaxrs/jaxrs-deltaspike-data/pom.xml]: is this dependency required? The relevant specs/APIs should drag in transitively from the org.wildfly.swarm:* dependencies, I think?

**Comment by bobmcwhirter** [comment]
Is there some integration test we could add to this to ensure/validate that it works and continues to work? Other examples have some tests that use WebDriver/Arq-Drone to check a few URLs to at least ensure stuff is functioning moderately well.

**Comment by bobmcwhirter** [comment]
retest this please

**Comment by bobmcwhirter** [comment]
@rafaelszp ignore the failure in the CI job, it's spurious.

**Comment by bobmcwhirter** [comment]
retest this please

**Comment by bobmcwhirter** [comment]
ok to test

**Comment by bobmcwhirter** [comment]
retest this please

**Comment by bobmcwhirter** [comment]
retest this please

**Comment by bobmcwhirter** [comment]
retest this please

**Comment by bobmcwhirter** [comment]
Yah, the way our integration tests work, using maven-failsafe-plugin, we actually run the -swarm.jar (either as uberjar or via mvn wildfly-swarm:run), so the tests are purely client-side.  The only arquillian usage is the WebDriver/Drone bits, to allow for testing externally using phantomjs.

**Comment by rafaelszp** [comment]
Is there anything else I should do?

**Pull request merged**